framework/lib/components/combo-picker/combo-picker.tsx

```diff
@@ -87,10 +90,24 @@ export const ComboPicker = ({
   }, [ enableSelectInput ])
 
   useEffect(() => {
-    if (!is(Number, inputValue) && defaultToZeroIfEmpty) {
-      handleInputChange(clamp(min, max, 0))
+    const needsToCorrectOption = !selectOption
```

Comment:
mby use isNil here, just to be a bit more clear cause javascript type conversions can be a bit confusing

Reply:
There is the potential case in which NaN could be passed as the inputValue, and that would not be considered nil, that's why I considered checking for Number and exclude any other case

Reply:
I meant for selectOption

Reply:
can skip this